search/tests/validator/stacValidator.spec.js

```diff
@@ -2,18 +2,25 @@ const { validateStac } = require('../../lib/validator');
 
 describe('stacValidator', () => {
   describe('validateStac', () => {
-    const validStacFeature = {
-      stac_version: '1.0.0-beta.1',
-      id: '1',
-      description: 'description',
-      links: []
+    const validStacFeatureCollection = {
+      features: [
+        {
+          stac_version: '0.8.0',
```

Comment:
@DrewE84 ,

Does this need to be updated to 1.0.0-beta.1 ?

Reply:
@DrewE84 ,

Disregard the above.  After talking to @jasongilman , it sounds like we'll be deprecating `stacValidator.js`